tcc/src/main/java/io/seata/rm/tcc/interceptor/ActionInterceptorHandler.java

```diff
@@ -82,9 +86,25 @@
         ret.put(Constants.TCC_METHOD_ARGUMENTS, arguments);
         //the final result
         ret.put(Constants.TCC_METHOD_RESULT, targetCallback.execute());
+        doUpdateTccActionLogStore(actionContext, ret);
         return ret;
     }
 
+    private void doUpdateTccActionLogStore(BusinessActionContext actionContext, Map<String, Object> ret) {
+        try {
+            //update branch record
+            actionContext.addActionContext(Constants.TCC_METHOD_RESULT,ret.get(Constants.TCC_METHOD_RESULT));
```

Comment:
TCC_METHOD_RESULT just a record.  Is this a little bit more customizable？ It is cost of an extra RPC time.
I'm not sure it's worth it

Reply:
Share context is the user's requirement.

Context sharing: in parameter, in execution, return value. The best way is to transmit phase 1 ‘in execution’ parameters to phase 2，such as orderId. But 'in execution' parameters in-branch server. 

A compromise way is action RPC server return ‘orderId', then pass to phase 2.  If we do not provide context sharing, users need to associate XID through third-party storage during method execution to share context. This kind of RPC takes more time and also has a bad experience.

 Later, if the same database mode is provided. The RPC costs will be greatly reduced.